### ISSUE TITLE
Project alive?

### ISSUE THREAD

**Issue by nx10** [octobanana/belle#13]
Is this still being worked on?

**Comment by octobanana** [comment]
Yes! It's just been a bit neglected. I've now got it working with Boost 1.72. Some more testing needs to be done to make sure it all works properly, and a few things need cleaning up before a new release is made. The next project I'll be working on will need this library to work with the newer Boost libs, so you can expect a new release soon!

**Comment by nx10** [comment]
Thank you for the response. That's great news!

**Comment by nx10** [comment]
How is it going @octobanana ? Any update when the next version will release?

**Comment by biddisco** [comment]
Any news on updates?
